src/test/resources/org/jnode/fs/ntfs/libfsntfs/fsntfs_test_compression_uncompressed_data1.bin

```diff
@@ -0,0 +1,166 @@
+		   GNU LESSER GENERAL PUBLIC LICENSE
+                       Version 3, 29 June 2007
```

Comment:
I don't think you can put LGPLv3 code in there, since this model is Apache licensed, which is compatible.